### ISSUE TITLE
Error using SQS list-queue-tags

### ISSUE THREAD

**Issue by rclmenezes** [localstack/localstack#551]
**Problem**
`aws sqs list-queue-tags` doesn't work.

**Steps to repro**
1) Make a queue:

```
$ aws --endpoint-url=http://localhost:4576 sqs create-queue --queue-name test
{
    "QueueUrl": "http://localhost:4576/queue/test"
}
```

2) Get the queue url:

```
$ aws --endpoint-url=http://localhost:4576 sqs list-queues
{
    "QueueUrls": [
        "http://localhost:4576/queue/test"
    ]
}
```

3) Try running `list-queue-tags`:

```
$ aws --endpoint-url=http://localhost:4576 sqs list-queue-tags --queue-url http://localhost:4576/queue/test

Unable to parse response (syntax error: line 1, column 0), invalid XML received:
The requested resource could not be found.
```

**Comments**
Unfortunately, this prevents us from using [terraform](https://www.terraform.io/) with localstack, since it always tries to hit `list-queue-tags`.

**Comment by whummer** [comment]
Thanks for reporting @rclmenezes . We're currently using [ElasticMQ](https://github.com/adamw/elasticmq) as the SQS backend, as it provides an implementation that is more in line with the AWS API (at least that was the case a few months ago). This seems like an upstream bug in ElasticMQ, we should probably file an issue in https://github.com/adamw/elasticmq. 

Meanwhile, we can create a temporary patch to fix the issue on our end. [`ProxyListenerSQS`](https://github.com/localstack/localstack/blob/master/localstack/services/sqs/sqs_listener.py#L13) would be a good starting point: we could check for `list-queue-tags` invocations and simply return an empty result for now. Would you be able to look into that? Thanks

**Comment by rclmenezes** [comment]
Hi,

I'm trying to work on this, but I'm unable to run tests:

```
$ make test
make lint && \
		(. .venv/bin/activate; DEBUG= PYTHONPATH=`pwd` nosetests --with-coverage --logging-level=WARNING --nocapture --no-skip --exe --cover-erase --cover-tests --cover-inclusive --cover-package=localstack --with-xunit --exclude='.venv.*' --ignore-files='lambda_python3.py' .)
(. .venv/bin/activate; flake8 --inline-quotes=single --show-source --max-line-length=120 --ignore=E128 --exclude=node_modules,.venv*,dist .)
Starting mock API Gateway (http port 4567)...
Starting mock DynamoDB (http port 4569)...
Starting mock SES (http port 4579)...
Starting mock Kinesis (http port 4568)...
Starting mock Redshift (http port 4577)...
Starting mock S3 (http port 4572)...
Starting mock CloudWatch (http port 4582)...
Starting mock CloudFormation (http port 4581)...
Starting mock SSM (http port 4583)...
Starting mock SQS (http port 4576)...
Starting local Elasticsearch (http port 4571)...
Starting mock SNS (http port 4575)...
Starting mock DynamoDB Streams service (http port 4570)...
Starting mock Firehose service (http port 4573)...
Starting mock Route53 (http port 4580)...
Starting mock ES service (http port 4578)...
Starting mock Lambda service (http port 4574)...
ERROR: Error forwarding request: HTTPConnectionPool(host='127.0.0.1', port=4564): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x112ad9250>: Failed to establish a new connection: [Errno 61] Connection refused',)) Traceback (most recent call last):
  File "/Users/rodrigo/code/localstack/localstack/services/generic_proxy.py", line 201, in forward
    headers=forward_headers)
  File "/Users/rodrigo/.virtualenvs/localstack/lib/python2.7/site-packages/requests/api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "/Users/rodrigo/.virtualenvs/localstack/lib/python2.7/site-packages/requests/api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "/Users/rodrigo/.virtualenvs/localstack/lib/python2.7/site-packages/requests/sessions.py", line 508, in request
    resp = self.send(prep, **send_kwargs)
  File "/Users/rodrigo/.virtualenvs/localstack/lib/python2.7/site-packages/requests/sessions.py", line 618, in send
    r = adapter.send(request, **kwargs)
  File "/Users/rodrigo/.virtualenvs/localstack/lib/python2.7/site-packages/requests/adapters.py", line 508, in send
    raise ConnectionError(e, request=request)
ConnectionError: HTTPConnectionPool(host='127.0.0.1', port=4564): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x112ad9250>: Failed to establish a new connection: [Errno 61] Connection refused',))
```

Running this on MacOS. Any advice?

**Comment by whummer** [comment]
Looks like there is an issue with the local DynamoDB (`4564` is the port for the internal DynamoDB server). Can you try enabling debug log output like this:
```
DEBUG=1 make test
```
Also, what is the output of this command?
```
ls -la localstack/infra/dynamodb
```
Thanks

**Comment by rclmenezes** [comment]
```
$ DEBUG=1 make test
make lint && \
		(. .venv/bin/activate; DEBUG=1 PYTHONPATH=`pwd` nosetests --with-coverage --logging-level=WARNING --nocapture --no-skip --exe --cover-erase --cover-tests --cover-inclusive --cover-package=localstack --with-xunit --exclude='.venv.*' --ignore-files='lambda_python3.py' .)
(. .venv/bin/activate; flake8 --inline-quotes=single --show-source --max-line-length=120 --ignore=E128 --exclude=node_modules,.venv*,dist .)
Starting mock API Gateway (http port 4567)...
Starting mock DynamoDB (http port 4569)...
Starting mock SES (http port 4579)...
Starting mock Kinesis (http port 4568)...
Starting mock Redshift (http port 4577)...
Starting mock S3 (http port 4572)...
Starting mock CloudWatch (http port 4582)...
Starting mock CloudFormation (http port 4581)...
Starting mock SSM (http port 4583)...
Error: Unable to access jarfile DynamoDBLocal.jar
Starting mock SQS (http port 4576)...
Starting local Elasticsearch (http port 4571)...
Error: Unable to access jarfile /Users/rodrigo/code/localstack/localstack/infra/elasticmq/elasticmq-server.jar
Starting mock SNS (http port 4575)...
Starting mock DynamoDB Streams service (http port 4570)...
Starting mock Firehose service (http port 4573)...
Starting mock Route53 (http port 4580)...
Starting mock ES service (http port 4578)...
Starting mock Lambda service (http port 4574)...
```

Looks like DynamoDBLocal isn't installed and neither is ElasticMQ?

```
$ ls -la localstack/infra/dynamodb
total 0
drwxr-xr-x  2 rodrigo  staff   68 Jan 17 17:25 .
drwxr-xr-x  6 rodrigo  staff  204 Jan 17 17:47 ..
```

Am I supposed to install them separately?

**Comment by rclmenezes** [comment]
Ran `make clean` and then `make install`. No dice:

```
$ ls ~/code/localstack/localstack/infra/elasticmq/
$ ls ~/code/localstack/localstack/infra/
amazon-kinesis-client		elasticmq			localstack-utils-fat.jar
```

**Comment by rclmenezes** [comment]
Ran `python localstack/services/install.py libs` and that worked 🤷‍♂️

**Comment by rclmenezes** [comment]
As an aside, we probably need to add `elasticmq` to `make clean` :)

https://github.com/localstack/localstack/pull/572

**Comment by rclmenezes** [comment]
Make `list-queue-tags` return a valid empty response:

https://github.com/localstack/localstack/pull/573

**Comment by whummer** [comment]
Workaround has been merged in #573 - closing.
